README.md

```diff
@@ -1,23 +1,24 @@
-# cc @sseraphini
+# cc @sseraphini monorepo
 
 [cc @sseraphini article](https://sibelius.substack.com/p/cc-sseraphini)
 
-## Setting up the environment variables
+This monorepo uses [Yarn Berry](https://github.com/yarnpkg/berry) for repository orchestration.
 
-Before running the project it is needed to create a .env file in the root of the project similar as is in .env.example, but replacing the variables to the correct ones.
+To use Yarn's latest version, run:
 
-### TWITTER_BEARER_TOKEN
+- `yarn set version berry`
 
-This is the Twitter API Bearer Token. You can get a new one with the following steps:
 
-- Access https://developer.twitter.com/en/portal/dashboard
-- Setup a new developer account. It can be a twitter account that you already have
-- Create a new project
-- Create an app inside the project
-- After that you will have the API Key, the Bearer Token and the Access Token. In this case we will only need the Bearer Token. After this just add it to the .env file and you are all set.
+## Plugins
 
-## How to run locally
+To make workspaces a more pleasing experience install these plugins:
 
-```
-yarn dev
-```
+- `yarn plugin import workspace-tools` - https://github.com/yarnpkg/berry/tree/master/packages/plugin-workspace-tools
+
+## Running Apps
+
+You can find all apps inside `apps/` and the convention `dev`, `test`, `lint` in all `package.json`.
+
+Because of that, you can use Yarn's `workspace` command with the `name` field from `package.json`:
+
+- `yarn workspace @ccsseraphini/web run dev` - To start the `web` project locally
```

Comment:
To install and start the `web` project locally, you can run the following command:

```shell
yarn workspace @ccsseraphini/web install && yarn workspace @ccsseraphini/web run dev
```

Reply:
with workspaces tools already installed `.yarnrc.yml`, you can do:

```
yarn workspaces foreach install
```

and it will run all the install for each project inside the monorepo,

what i usually do is add a `setup:local` script in the root `package.json` with all requirement commands to get the repo ready, the dx in the end is something like:

```bash
git clone x
yarn run setup:local // => this should run workspaces foreach install + create .env files + fetch creds or generate any required files etc, using zx from google
```

Reply:
this option makes total sense, thanks for sharing I didn't even know about it (I didn't have many opportunities to work with monorepos either) o/

I think it's cool that we add a comment somewhere about how to install them (all packages/repos) with one command or just a specific one without having to navigate inside the directory and run the command manually 🤔 

> Do you think we can add the setup command in the parent repository package?